com.github.rajsolai.response.yml

```diff
@@ -0,0 +1,22 @@
+app-id: com.github.rajsolai.response
+
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+base: io.elementary.BaseApp
+base-version: 'juno-20.08'
+sdk: org.gnome.Sdk
+command: com.github.rajsolai.response
+
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--share=network'
```

Comment:
you probably want to keep dri permission here, especially that you're using webkit

Reply:
Ohh... I tested on my local computer it worked fine btw app does'nt save any cookies with webkit.